### ISSUE TITLE
Add option to disable customer input.

### ISSUE THREAD

**Issue by mintplugins** [easydigitaldownloads/edd-invoices#28]
A customer has requested that an option be available to disable customer input on invoices and that, instead, Invoices be automatically generated with a single click. 

We could add an option in EDD Invoice settings to disable customer input and use payment details instead.

**Comment by pippinsplugins** [comment]
They should use PDF Invoices instead then. That's precisely what it is for.

If we end up doing this, I'd vote it be done through the merging of PDF Invoices and EDD Invoices.

**Comment by mintplugins** [comment]
@pippinsplugins True. In this case, this customer was aware of the PDF Invoices extension:

`I like the idea it is not server heavy like the pdf generator extension but the customer inputting details to generate the receipt is a bit confusing to me.`
